app/streaming/video/ffmpeg.cpp

```diff
@@ -190,6 +194,15 @@ void FFmpegVideoDecoder::reset()
 
 bool FFmpegVideoDecoder::createFrontendRenderer(PDECODER_PARAMETERS params)
 {
+#ifdef HAVE_EGL
+    if (m_BackendRenderer->canExportEGL()) {
+        m_FrontendRenderer = new EGLRenderer(m_BackendRenderer);
+        if (m_FrontendRenderer->initialize(params)) {
+            return true;
+        }
+        delete m_FrontendRenderer;
+    }
```

Comment:
After thinking about this a bit, let's move this into the `!m_BackendRenderer->isDirectRenderingSupported()` case below. That will reduce the possible risk of regression to just the Wayland case which is already terrible today.

Once it has been in the wild for a bit, we can consider using EGL by default.